### DISCUSSION TITLE
2.3.11

### DISCUSSION THREAD

**Discussion by AmarnathCJD** [AmarnathCJD/gogram#96] [Announcements]
## What's Changed
* Update README.md by @MukundSinghRajput in https://github.com/AmarnathCJD/gogram/pull/94

**Full Changelog**

1acb973 - Merge branch 'master' of https://github.com/AmarnathCJD/gogram (AmarnathCJD)
cbfaf9d - Update ApiVersion to 181 (bug fixes) fix #88 v2.3.11 (AmarnathCJD)
ae12edb - Update formatting.go (Amarnath C)
88291ea - TLParser: Layer Updated (AmarnathCJD)
570ccee - Update newmessage.go (Amarnath C)
ead89ed - Merge pull request #94 from MukundSinghRajput/master (Amarnath C)
f8e26dc - Update newmessage.go (Mukund)
7a65d4e - Update README.md (Mukund)
b9bb5e3 - back in buisness; (xAmarnath)
7209a61 - Add CloseChannelWithoutPanic function to mtproto.go (GithubAI)
761494b - bug fixes, Add shorthandle for client.SendPoll (GithubAI)
f66f155 - ci: update gofmt.yml and tlparser.yml bump up git-actions versions (GithubAI)
feae39b - Update gofmt workflow and schedule in .github/workflows/gofmt.yml Update cron schedule in .github/workflows/tlparser.yml (GithubAI)
02d7ebc - TLParser: Layer Updated (AmarnathCJD)
53ac9c3 - Add support for new tags in telegram/formatting.go (GithubAI)

<hr /><em>This discussion was created from the release <a href='https://github.com/AmarnathCJD/gogram/releases/tag/2.3.11'>2.3.11</a>.</em>

**Comment by MukundSinghRajput** [comment]
Good
